demos/instantsearchjs/ecommerce/src/app.js

```diff
@@ -1,103 +1,106 @@
-/* global instantsearch */
+/* global algoliasearch, instantsearch */
 
-import { hitTemplate } from "./helpers";
+import { hitTemplate } from './helpers'
 
 const search = instantsearch({
-  appId: "B1G2GM9NG0",
-  apiKey: "aadef574be1f9252bb48d4ea09b5cfe5",
-  indexName: "demo_ecommerce",
-  searchParameters: {
+  indexName: 'demo_ecommerce',
+  searchClient: algoliasearch('B1G2GM9NG0', 'aadef574be1f9252bb48d4ea09b5cfe5'),
+})
+
+search.addWidget(
+  instantsearch.widgets.configure({
     hitsPerPage: 5,
-    attributesToSnippet: ["description:24"],
-    snippetEllipsisText: " [...]"
-  }
-});
+    attributesToSnippet: ['description:24'],
+    snippetEllipsisText: ' [...]',
+  })
+)
 
 // Uncomment the following widget to add hits list.
 
 /* search.addWidget(
   instantsearch.widgets.hits({
-    container: "#hits",
+    container: '#hits',
     templates: {
-      empty: "No results.",
-      item: function(hit) {
-        return hitTemplate(hit);
-      }
-    }
+      empty: 'No results.',
+      item: hitTemplate,
+    },
   })
-); */
+) */
```

Comment:
I wonder if we should use `//` to comment widgets instead of `/*` since we can't uncomment them easily on CSB (you have to manually delete `/*` and `*/`).

Reply:
I'm mainly thinking about folks that don't know their keyboard shortcuts or have non-qwerty keyboard layouts that won't allow them to `⌘ + /`. In this case then it's easier to remove 4 characters than comments on every line.

Reply:
Good point!